railties/lib/rails/templates/rails/mailers/email.html.erb

```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
 <html><head>
+<title>Mailer Preview</title>
```

Comment:
The index page set mailer name to tile. 
https://github.com/rails/rails/blob/782d28030730c14658a7cd66ee81e7f16c0a12cd/railties/lib/rails/mailers_controller.rb#L23   

So I think it is better to individual pages have mailer name and actions name in title.

Reply:
@y-yagi good suggestion. I've made that change.